src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -126,14 +126,14 @@ export class GwtCallback extends EventEmitter {
 
     ipcMain.on('desktop_browse_url', (event, url: string) => {
 
-      // shell.openExternal() doesn't handle file URIs containining non-ASCII characters
+      // shell.openExternal() seems unreliable on Windows
```

Comment:
Looks to be tracked (more or less) via https://github.com/electron/electron/issues/31347, maybe add that in the comment in case we want to revisit someday and see if it has been fixed?